2021-02-14 09:25:51.7644|ERROR|TiendaJapSoftware.Server.Repositorio.RepositorioGeneral|Se produjo un error al dar de alta nuestro usuario en nuestra BBDD :System.Data.SqlClient.SqlException (0x80131904): El usuario especificado ya existe japsoftware@gmail.com
   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString)
   at System.Data.SqlClient.SqlCommand.CompleteAsyncExecuteReader()
   at System.Data.SqlClient.SqlCommand.EndExecuteNonQueryInternal(IAsyncResult asyncResult)
   at System.Data.SqlClient.SqlCommand.EndExecuteNonQuery(IAsyncResult asyncResult)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location ---
   at TiendaJapSoftware.Server.Repositorio.RepositorioGeneral.AltaUsuario(Usuario u) in D:\Videos\Ejemplos codigo\TiendaJapSoftware\TiendaJapSoftware\Server\Repositorio\RepositorioGeneral.cs:line 95
ClientConnectionId:acbbc902-07e1-4e60-8103-0e968c73c03b
Error Number:50000,State:1,Class:16
2021-02-14 09:32:24.0430|ERROR|TiendaJapSoftware.Server.Repositorio.RepositorioGeneral|Se produjo un error al dar de alta nuestro usuario en nuestra BBDD :System.Data.SqlClient.SqlException (0x80131904): El usuario especificado ya existe japsoftware@gmail.com
   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString)
   at System.Data.SqlClient.SqlCommand.CompleteAsyncExecuteReader()
   at System.Data.SqlClient.SqlCommand.EndExecuteNonQueryInternal(IAsyncResult asyncResult)
   at System.Data.SqlClient.SqlCommand.EndExecuteNonQuery(IAsyncResult asyncResult)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location ---
   at TiendaJapSoftware.Server.Repositorio.RepositorioGeneral.AltaUsuario(Usuario u) in D:\Videos\Ejemplos codigo\TiendaJapSoftware\TiendaJapSoftware\Server\Repositorio\RepositorioGeneral.cs:line 95
ClientConnectionId:acbbc902-07e1-4e60-8103-0e968c73c03b
Error Number:50000,State:1,Class:16
2021-02-14 09:36:07.7962|ERROR|TiendaJapSoftware.Server.Repositorio.RepositorioGeneral|Se produjo un error al dar de alta nuestro usuario en nuestra BBDD :System.Data.SqlClient.SqlException (0x80131904): El usuario especificado ya existe japsoftware@gmail.com
   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString)
   at System.Data.SqlClient.SqlCommand.CompleteAsyncExecuteReader()
   at System.Data.SqlClient.SqlCommand.EndExecuteNonQueryInternal(IAsyncResult asyncResult)
   at System.Data.SqlClient.SqlCommand.EndExecuteNonQuery(IAsyncResult asyncResult)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location ---
   at TiendaJapSoftware.Server.Repositorio.RepositorioGeneral.AltaUsuario(Usuario u) in D:\Videos\Ejemplos codigo\TiendaJapSoftware\TiendaJapSoftware\Server\Repositorio\RepositorioGeneral.cs:line 95
ClientConnectionId:33351b2e-d1c1-4659-bc12-b17652d76b69
Error Number:50000,State:1,Class:16
2021-02-14 10:04:32.6684|ERROR|TiendaJapSoftware.Server.Repositorio.RepositorioGeneral|Se produjo un error al dar de alta nuestro usuario en nuestra BBDD :System.Data.SqlClient.SqlException (0x80131904): El usuario especificado ya existe japsoftware@gmail.com
   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString)
   at System.Data.SqlClient.SqlCommand.CompleteAsyncExecuteReader()
   at System.Data.SqlClient.SqlCommand.EndExecuteNonQueryInternal(IAsyncResult asyncResult)
   at System.Data.SqlClient.SqlCommand.EndExecuteNonQuery(IAsyncResult asyncResult)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location ---
   at TiendaJapSoftware.Server.Repositorio.RepositorioGeneral.AltaUsuario(Usuario u) in D:\Videos\Ejemplos codigo\TiendaJapSoftware\TiendaJapSoftware\Server\Repositorio\RepositorioGeneral.cs:line 95
ClientConnectionId:015d1f39-43f1-46ce-86e1-077e876ade84
Error Number:50000,State:1,Class:16
2021-02-14 10:30:09.5342|ERROR|TiendaJapSoftware.Server.Repositorio.RepositorioGeneral|Se produjo un error al dar de alta nuestro usuario en nuestra BBDD :System.Data.SqlClient.SqlException (0x80131904): El usuario especificado ya existe japsoftware@gmail.com
   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString)
   at System.Data.SqlClient.SqlCommand.CompleteAsyncExecuteReader()
   at System.Data.SqlClient.SqlCommand.EndExecuteNonQueryInternal(IAsyncResult asyncResult)
   at System.Data.SqlClient.SqlCommand.EndExecuteNonQuery(IAsyncResult asyncResult)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location ---
   at TiendaJapSoftware.Server.Repositorio.RepositorioGeneral.AltaUsuario(Usuario u) in D:\Videos\Ejemplos codigo\TiendaJapSoftware\TiendaJapSoftware\Server\Repositorio\RepositorioGeneral.cs:line 95
ClientConnectionId:0a7ce024-2960-4235-9467-89bdd1ab8717
Error Number:50000,State:1,Class:16
2021-02-14 10:32:55.8939|ERROR|TiendaJapSoftware.Server.Repositorio.RepositorioGeneral|Se produjo un error al dar de alta nuestro usuario en nuestra BBDD :System.Data.SqlClient.SqlException (0x80131904): El usuario especificado ya existe japsoftware@gmail.com
   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString)
   at System.Data.SqlClient.SqlCommand.CompleteAsyncExecuteReader()
   at System.Data.SqlClient.SqlCommand.EndExecuteNonQueryInternal(IAsyncResult asyncResult)
   at System.Data.SqlClient.SqlCommand.EndExecuteNonQuery(IAsyncResult asyncResult)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location ---
   at TiendaJapSoftware.Server.Repositorio.RepositorioGeneral.AltaUsuario(Usuario u) in D:\Videos\Ejemplos codigo\TiendaJapSoftware\TiendaJapSoftware\Server\Repositorio\RepositorioGeneral.cs:line 95
ClientConnectionId:df0e8e0c-60f7-405b-acb3-339af7ee3039
Error Number:50000,State:1,Class:16
2021-02-14 10:34:04.1048|ERROR|TiendaJapSoftware.Server.Repositorio.RepositorioGeneral|Se produjo un error al dar de alta nuestro usuario en nuestra BBDD :System.Data.SqlClient.SqlException (0x80131904): El usuario especificado ya existe japsoftware@gmail.com
   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString)
   at System.Data.SqlClient.SqlCommand.CompleteAsyncExecuteReader()
   at System.Data.SqlClient.SqlCommand.EndExecuteNonQueryInternal(IAsyncResult asyncResult)
   at System.Data.SqlClient.SqlCommand.EndExecuteNonQuery(IAsyncResult asyncResult)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location ---
   at TiendaJapSoftware.Server.Repositorio.RepositorioGeneral.AltaUsuario(Usuario u) in D:\Videos\Ejemplos codigo\TiendaJapSoftware\TiendaJapSoftware\Server\Repositorio\RepositorioGeneral.cs:line 95
ClientConnectionId:df0e8e0c-60f7-405b-acb3-339af7ee3039
Error Number:50000,State:1,Class:16
2021-02-14 11:40:05.6028|ERROR|TiendaJapSoftware.Server.Repositorio.RepositorioGeneral|Se produjo un error al dar de alta nuestro usuario en nuestra BBDD :System.Data.SqlClient.SqlException (0x80131904): El usuario especificado ya existe japsoftware@gmail.com
   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString)
   at System.Data.SqlClient.SqlCommand.CompleteAsyncExecuteReader()
   at System.Data.SqlClient.SqlCommand.EndExecuteNonQueryInternal(IAsyncResult asyncResult)
   at System.Data.SqlClient.SqlCommand.EndExecuteNonQuery(IAsyncResult asyncResult)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location ---
   at TiendaJapSoftware.Server.Repositorio.RepositorioGeneral.AltaUsuario(Usuario u) in D:\Videos\Ejemplos codigo\TiendaJapSoftware\TiendaJapSoftware\Server\Repositorio\RepositorioGeneral.cs:line 100
ClientConnectionId:b31d1a2f-c352-43a6-8351-4ab4a74027fb
Error Number:50000,State:1,Class:16
2021-02-14 13:34:26.5360|ERROR|TiendaJapSoftware.Server.Repositorio.RepositorioGeneral|Se produjo un error al iniciar sesión en el controlador: System.NullReferenceException: Object reference not set to an instance of an object.
   at TiendaJapSoftware.Server.Repositorio.RepositorioGeneral.ValidarUsuario(String email) in D:\Videos\Ejemplos codigo\TiendaJapSoftware\TiendaJapSoftware\Server\Repositorio\RepositorioGeneral.cs:line 149
   at TiendaJapSoftware.Server.Controllers.TiendaController.ValidarUsuario(UsuarioLogIn login) in D:\Videos\Ejemplos codigo\TiendaJapSoftware\TiendaJapSoftware\Server\Controllers\TiendaController.cs:line 128
2021-02-14 13:34:31.2644|ERROR|Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware|An unhandled exception has occurred while executing the request.
2021-02-14 13:39:42.5235|ERROR|TiendaJapSoftware.Server.Repositorio.RepositorioGeneral|Se produjo un error al iniciar sesión en el controlador: System.Exception: Credenciales no validas
   at TiendaJapSoftware.Server.Controllers.TiendaController.ValidarUsuario(UsuarioLogIn login) in D:\Videos\Ejemplos codigo\TiendaJapSoftware\TiendaJapSoftware\Server\Controllers\TiendaController.cs:line 131
2021-02-14 13:39:42.6099|ERROR|Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware|An unhandled exception has occurred while executing the request.
2021-02-14 13:39:44.1427|ERROR|TiendaJapSoftware.Server.Repositorio.RepositorioGeneral|Se produjo un error al iniciar sesión en el controlador: System.Exception: Credenciales no validas
   at TiendaJapSoftware.Server.Controllers.TiendaController.ValidarUsuario(UsuarioLogIn login) in D:\Videos\Ejemplos codigo\TiendaJapSoftware\TiendaJapSoftware\Server\Controllers\TiendaController.cs:line 131
2021-02-14 13:39:44.1832|ERROR|Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware|An unhandled exception has occurred while executing the request.
2021-02-14 13:39:48.8391|ERROR|TiendaJapSoftware.Server.Repositorio.RepositorioGeneral|Se produjo un error al iniciar sesión en el controlador: System.Exception: Credenciales no validas
   at TiendaJapSoftware.Server.Controllers.TiendaController.ValidarUsuario(UsuarioLogIn login) in D:\Videos\Ejemplos codigo\TiendaJapSoftware\TiendaJapSoftware\Server\Controllers\TiendaController.cs:line 131
2021-02-14 13:39:48.8791|ERROR|Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware|An unhandled exception has occurred while executing the request.
2021-02-14 13:39:52.2915|ERROR|TiendaJapSoftware.Server.Repositorio.RepositorioGeneral|Se produjo un error al iniciar sesión en el controlador: System.Exception: Credenciales no validas
   at TiendaJapSoftware.Server.Controllers.TiendaController.ValidarUsuario(UsuarioLogIn login) in D:\Videos\Ejemplos codigo\TiendaJapSoftware\TiendaJapSoftware\Server\Controllers\TiendaController.cs:line 131
2021-02-14 13:39:52.3287|ERROR|Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware|An unhandled exception has occurred while executing the request.
2021-02-14 13:40:03.9291|ERROR|TiendaJapSoftware.Server.Repositorio.RepositorioGeneral|Se produjo un error al iniciar sesión en el controlador: System.Exception: Credenciales no validas
   at TiendaJapSoftware.Server.Controllers.TiendaController.ValidarUsuario(UsuarioLogIn login) in D:\Videos\Ejemplos codigo\TiendaJapSoftware\TiendaJapSoftware\Server\Controllers\TiendaController.cs:line 131
2021-02-14 13:40:03.9648|ERROR|Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware|An unhandled exception has occurred while executing the request.
2021-02-14 13:41:25.4458|ERROR|TiendaJapSoftware.Server.Repositorio.RepositorioGeneral|Se produjo un error al iniciar sesión en el controlador: System.Exception: Credenciales no validas
   at TiendaJapSoftware.Server.Controllers.TiendaController.ValidarUsuario(UsuarioLogIn login) in D:\Videos\Ejemplos codigo\TiendaJapSoftware\TiendaJapSoftware\Server\Controllers\TiendaController.cs:line 131
2021-02-14 13:41:25.5274|ERROR|Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware|An unhandled exception has occurred while executing the request.
2021-02-14 13:41:27.6673|ERROR|TiendaJapSoftware.Server.Repositorio.RepositorioGeneral|Se produjo un error al iniciar sesión en el controlador: System.Exception: Credenciales no validas
   at TiendaJapSoftware.Server.Controllers.TiendaController.ValidarUsuario(UsuarioLogIn login) in D:\Videos\Ejemplos codigo\TiendaJapSoftware\TiendaJapSoftware\Server\Controllers\TiendaController.cs:line 131
2021-02-14 13:41:27.7187|ERROR|Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware|An unhandled exception has occurred while executing the request.
2021-02-14 13:41:41.5687|ERROR|TiendaJapSoftware.Server.Repositorio.RepositorioGeneral|Se produjo un error al iniciar sesión en el controlador: System.Exception: Credenciales no validas
   at TiendaJapSoftware.Server.Controllers.TiendaController.ValidarUsuario(UsuarioLogIn login) in D:\Videos\Ejemplos codigo\TiendaJapSoftware\TiendaJapSoftware\Server\Controllers\TiendaController.cs:line 131
2021-02-14 13:41:41.6045|ERROR|Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware|An unhandled exception has occurred while executing the request.
2021-02-14 13:41:59.2609|ERROR|TiendaJapSoftware.Server.Repositorio.RepositorioGeneral|Se produjo un error al iniciar sesión en el controlador: System.Exception: Credenciales no validas
   at TiendaJapSoftware.Server.Controllers.TiendaController.ValidarUsuario(UsuarioLogIn login) in D:\Videos\Ejemplos codigo\TiendaJapSoftware\TiendaJapSoftware\Server\Controllers\TiendaController.cs:line 131
2021-02-14 13:41:59.3003|ERROR|Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware|An unhandled exception has occurred while executing the request.
2021-02-14 15:18:38.3650|ERROR|TiendaJapSoftware.Server.Repositorio.RepositorioGeneral|Se produjo un error al dar de alta nuestro usuario en nuestra BBDD :System.Data.SqlClient.SqlException (0x80131904): El usuario especificado ya existe japsoftware7@gmail.com
   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString)
   at System.Data.SqlClient.SqlCommand.CompleteAsyncExecuteReader()
   at System.Data.SqlClient.SqlCommand.EndExecuteNonQueryInternal(IAsyncResult asyncResult)
   at System.Data.SqlClient.SqlCommand.EndExecuteNonQuery(IAsyncResult asyncResult)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location ---
   at TiendaJapSoftware.Server.Repositorio.RepositorioGeneral.AltaUsuario(Usuario u) in D:\Videos\Ejemplos codigo\TiendaJapSoftware\TiendaJapSoftware\Server\Repositorio\RepositorioGeneral.cs:line 99
ClientConnectionId:178fe35d-3ca3-4ef7-a093-f49b4ebf5a85
Error Number:50000,State:1,Class:16
